src/api/HydroCodeLookup.cs

```diff
@@ -4,61 +4,50 @@ public static class HydroCodeLookup
 {
```

Comment:
REQ: Die Klasse muss ausführlich dokumentiert werden, da man aktuell nicht weiss wofür die Zahlen in den Dictionaries stehen.